x-pack/plugins/ess_security/public/common/components/landing_cards/index.tsx

```diff
@@ -73,13 +84,25 @@ export const LandingCards = memo(() => {
   );
 
   const href = useMemo(() => prepend(addIntegrationsUrl), [prepend, addIntegrationsUrl]);
+
   return (
-    <EuiFlexGroup data-test-subj="siem-landing-page" direction="column" gutterSize="l">
+    <EuiFlexGroup
+      data-test-subj="siem-landing-page"
+      direction="column"
+      gutterSize="l"
+      css={css`
+        padding: ${euiTheme.size.l};
+      `}
```

Comment:
I mean to remove this `css` prop

Reply:
It would be probably better to wrap this whole page in a `<EuiPageTemplate.Section paddingSize="l">` than adding an explicit rule via CSS to the FlexGroup.